tests/cpp/test_host_ir_jit.cpp

```diff
@@ -34,28 +33,20 @@ TEST_F(HostIrJitTest, Allocate) {
   hic->addInput(hic_in);
   hic->addOutput(hic_out);
 
-  // Adjust the number of allocates and calls to each allocate
-  int num_allocates = 10;
-  int num_calls_per_allocate = 10;
-  for (int i = 0; i < num_allocates; i++) {
-    auto* allocate =
-        IrBuilder::create<kir::Allocate>(hic_out, MemoryType::Global);
-    hic->pushBackTopLevelExprs(allocate);
-  }
+  hic->pushBackTopLevelExprs(hic_out->definition());
+
+  HostIrJit jit(std::move(hic));
+  auto options = at::TensorOptions().dtype(at::kFloat).device(at::kCUDA, 0);
+  int run_iterations = 10;
+  for (int i = 0; i < run_iterations; i++) {
+    int dim0 = std::rand() % 100 + 1;
+    int dim1 = std::rand() % 100 + 1;
+    at::Tensor t0 = at::randn({dim0, dim1}, options);
```

Comment:
I don't see much value of testing random values in a loop here. That doesn't cover more code paths because symbolic sizes are simply passed through as a payload. I'd simply pick a size, e.g. `{2, 3}`, and run just one iteration.